Add AUTH RULE notifications (#409)
neo4j/docs-status-codes: modules/ROOT/pages/notifications/all-notifications.adoc

======
=====

.Revoking a role from an auth rule that is not assigned to this role
[.tabbed-example]
=====
[.include-with-GQLSTATUS-code]
======
Command::
+
[source, cypher]
----
REVOKE ROLE admin, reader FROM AUTH RULE authrule
----

Returned GQLSTATUS code::
00N71

Returned status description::
note: successful completion - role or privilege not assigned.
`REVOKE ROLE reader FROM AUTH RULE authrule` has no effect.
The role or privilege is not assigned.

Suggestions for improvement::
Verify that this is the intended role and auth rule.
======
[.include-with-neo4j-code]
======
Command::
+
[source, cypher]
----
REVOKE ROLE admin, reader FROM AUTH RULE authrule
----

Title of the returned code::

> `REVOKE ROLE reader FROM AUTH RULE` has no effect.

Reviewer: comparing with the user one you're missing the rule name:
```suggestion
`REVOKE ROLE reader FROM AUTH RULE authrule` has no effect.
```

======
=====

> .Granting a role to an auth rule which already is assigned to that role

Reviewer: comparing with the user and privilege ones:
```suggestion
.Granting a role to an auth rule that already has that role
```
